### ISSUE TITLE
facebook app in dev mode

### ISSUE THREAD

**Issue by TBD** [hacktm15/happycity#1]
"App Not Setup: This app is still in development mode, and you don't have access to it. Switch to a registered test user or ask an app admin for permissions."

**Comment by dlucian** [comment]
Yeah, the Facebook app review & approval process takes time is pretty challenging.
